osbenchmark/worker_coordinator/runner.py

```diff
@@ -682,30 +682,38 @@ class ForceMerge(Runner):
     Runs a force merge operation against OpenSearch.
     """
 
+    PARAM_WAIT_FOR_COMPLETION = "wait_for_completion"
+
     async def __call__(self, opensearch, params):
-        # pylint: disable=import-outside-toplevel
-        import opensearchpy
         max_num_segments = params.get("max-num-segments")
         mode = params.get("mode")
         merge_params = self._default_kw_params(params)
         if max_num_segments:
             merge_params["max_num_segments"] = max_num_segments
         if mode == "polling":
-            complete = False
-            try:
-                request_context_holder.on_client_request_start()
-                await opensearch.indices.forcemerge(**merge_params)
-                request_context_holder.on_client_request_end()
-                complete = True
-            except opensearchpy.ConnectionTimeout:
-                pass
-            while not complete:
-                await asyncio.sleep(params.get("poll-period"))
-                tasks = await opensearch.tasks.list(params={"actions": "indices:admin/forcemerge"})
-                if len(tasks["nodes"]) == 0:
-                    # empty nodes response indicates no tasks
+            if params.get(self.PARAM_WAIT_FOR_COMPLETION):
```

Comment:
Will there ever be a case where `mode == "polling"` but `self.PARAM_WAIT_FOR_COMPLETION` is not in `params`? 

Even if `self.PARAM_WAIT_FOR_COMPLETION` is not found in params, `merge_params` will always set its key of `wait_for_completion` to be "false" anyways. If this is the case, we can remove this if statement and just have the logging statement aligned with the rest.

```
# Original 
        if mode == "polling":
            if params.get(self.PARAM_WAIT_FOR_COMPLETION):
                self.logger.warning(
                    "%s is set for polling. It will be updated to false", self.PARAM_WAIT_FOR_COMPLETION)
            merge_params[self.PARAM_WAIT_FOR_COMPLETION] = "false"

# New
       if mode == "polling":
                self.logger.warning(
                    "%s is set for polling. It will be updated to false", self.PARAM_WAIT_FOR_COMPLETION)
                merge_params[self.PARAM_WAIT_FOR_COMPLETION] = "false"
```

Reply:
Ack